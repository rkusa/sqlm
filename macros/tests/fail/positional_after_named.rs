fn main() {
    sqlm_macros::sql!("{one} {}", one = 1, 2);
}
