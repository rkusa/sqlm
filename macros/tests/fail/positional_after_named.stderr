error: positional arguments cannot follow named arguments
 --> tests/fail/positional_after_named.rs:2:44
  |
2 |     sqlm_macros::sql!("{one} {}", one = 1, 2);
  |                                            ^
